OWNERS

```diff
@@ -7,3 +7,12 @@ filters:
     approvers:
       - yanirq
       - ksimon1
+
```

Comment:
@beekhof  I would add you here to approvers list as well

Reply:
```suggestion
      - beekhof
```